bootstrap.js

```diff
@@ -230,8 +230,6 @@ function startup(aData, aReason) {
 function shutdown(aData, aReason) {
   // No need to do extra work here
   Log.debug("shutdown, aReason=", aReason);
-  // Load imports for the BOOTSTRAP_REASONS constants; should have happened before in most cases
-  loadImports();
```

Comment:
I don't think a new pdf version is too important...
What about the keyconfig-bug? If you knew the right Listener it could be fixed quite quickly...
Otherwise: MERGE! ;-)